dev-libs/json-c/json-c-0.14-r3.ebuild

```diff
@@ -13,7 +13,7 @@ SRC_URI="https://s3.amazonaws.com/json-c_releases/releases/${P}.tar.gz"
 LICENSE="MIT"
 SLOT="0/5"
 KEYWORDS="~alpha amd64 arm arm64 hppa ~ia64 ~mips ppc ppc64 s390 sparc x86 ~amd64-linux ~x86-linux ~ppc-macos"
-IUSE="doc static-libs threads"
+IUSE="cpu-flags-x86-rdrand doc static-libs threads"
```

Comment:
I wonder if this should be made to a legit USE expand for `cpu_flags_x86`?

Reply:
Hmm well we can add that. If it's just this one USE flag perhaps it can be like this

Reply:
Can you check whether **cpuid2cpuflags** detects it or not? And if not, _should_ it? (If it's taken into upstream and fixed there...)

Sorry I know this is a small edit, but trying to do it the right way so we don't have to be fixing it later on.

Reply:
I checked, and `cpuid2cpuflags`  doesn't detect the `rdrand` CPU flag, from what I checked that one is not supported in the package (https://github.com/mgorny/cpuid2cpuflags/blob/master/src/x86.c#L52)

Reply:
RDRAND is somewhat controversial these days, due to the bug in some (which?) AMD CPUs.

Reply:
ping @juippis Is it okay like this, or do we need to change something?